app/ide-desktop/lib/client/src/index.ts

```diff
@@ -92,6 +95,19 @@ class App {
         return { ...configParser.parseArgs(argsToParse), fileToOpen, urlToOpen }
     }
 
+    setProjectToOpenOnStartup(idOfProjectToOpen: string) {
```

Comment:
missing doc comment - shouldn't be a problem for now but i have an upcoming PR that (re?)enables the lint